StatusMonitor/StatusMonitor.cs

```diff
@@ -576,20 +576,20 @@ private void FuelPercentAndTime(decimal? fuelRemaining, decimal? fuelPerSecond,
             if (currentStatus.vehicle == Constants.VEHICLE_SHIP)
             {
                 Ship ship = EDDI.Instance.CurrentShip;
-                if (ship?.fueltanktotalcapacity != null && fuelRemaining != null)
+                if (ship?.fueltanktotalcapacity > 0)
```

Comment:
are we sure about discarding the null check on `fuelRemaining`?

Reply:
It was redundant. We already checked for null above (lines 571-574).
```cs
if (fuelRemaining is null)
{
    return;
}
```

Reply:
What about the ship being null?

Reply:
`ship?.fueltanktotalcapacity > 0` and `fuelPerSecond > 0` should handle those cases since comparing to a null value will always return false. 

Per https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/nullable-value-types:
>For the comparison operators <, >, <=, and >=, if one or both operands are null, the result is false; otherwise, the contained values of operands are compared. Do not assume that because a particular comparison (for example, <=) returns false, the opposite comparison (>) returns true. The following example shows that 10 is 
>* neither greater than or equal to null
>* nor less than null

```cs
int? a = 10;
Console.WriteLine($"{a} >= null is {a >= null}");
Console.WriteLine($"{a} < null is {a < null}");
Console.WriteLine($"{a} == null is {a == null}");
// Output:
// 10 >= null is False
// 10 < null is False
// 10 == null is False

int? b = null;
int? c = null;
Console.WriteLine($"null >= null is {b >= c}");
Console.WriteLine($"null == null is {b == c}");
// Output:
// null >= null is False
// null == null is True
```

Reply:
Thanks for the pointer :)